packages/cli/src/ExpressionEvalator.ts

```diff
@@ -0,0 +1,14 @@
+import config from '@/config';
+import { ErrorReporterProxy, ExpressionEvaluatorProxy } from 'n8n-workflow';
+
+export const initExpressionEvaluator = () => {
+	ExpressionEvaluatorProxy.setEvaluator(config.getEnv('expression.evaluator'));
+	ExpressionEvaluatorProxy.setDifferEnabled(config.getEnv('expression.reportDifference'));
+	ExpressionEvaluatorProxy.setDiffReporter((expr) => {
+		ErrorReporterProxy.warn('Expression difference', {
+			extra: {
+				expression: expr,
+			},
+		});
```

Comment:
Is there a way to avoid reporting differences that are only date-based to reduce noise in Sentry?

Reply:
I can't think of any that don't also include the chance of a false negative.